op-e2e/eip4844_test.go

```diff
@@ -0,0 +1,110 @@
+package op_e2e
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-service/client"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// TestSystem4844E2E runs the SystemE2E test with 4844 enabled on L1,
+// and active on the rollup in the op-batcher and verifier.
+func TestSystem4844E2E(t *testing.T) {
```

Comment:
Pulling in the full e2e 4844 test won't work until the 4844 batcher/verifier functionality lands.